package.json

```diff
@@ -1544,7 +1543,7 @@
 		"browsertest": "yarn run browsertest:preprocess && node ./dist/browser/test/runTests.js",
 		"test:preprocess-gql": "node scripts/preprocess-gql --in src/github/queries.gql --out out/src/github/queries.gql",
 		"test:preprocess-svg": "node scripts/preprocess-svg --in ../resources/ --out out/resources",
-		"update-dts": "pushd \"src/@types\" && npx vscode-dts main && npx vscode-dts dev && popd",
+		"update-dts": "cd \"src/@types\" && npx vscode-dts main && npx vscode-dts dev",
```

Comment:
Do we need to `cd ../../` at the end of `update-dts`?

Reply:
As far as I could tell, that was not necessary as the working directory is reset on each task